src/test/java/com/hivemq/cli/hivemq/behaviorpolicy/BehaviorPolicyCreateTaskTest.java

```diff
@@ -37,7 +37,7 @@
 public class BehaviorPolicyCreateTaskTest {
 
     private final @NotNull DataHubBehaviorPoliciesApi behaviorPoliciesApi = mock(DataHubBehaviorPoliciesApi.class);
```

Comment:
```suggestion
    private final @NotNull DataHubBehaviorPoliciesApi behaviorPoliciesApi = mock();
```

Also in other classes